apps/s_client.c

```diff
@@ -798,6 +801,31 @@ static int new_session_cb(SSL *S, SSL_SESSION *sess)
     return 0;
 }
 
+static void keylog_callback(const SSL *ssl, const char *line)
+{
+    BIO *keylog = BIO_new_file(keylog_file, "a");
+    char buffer[256];
```

Comment:
No, it just doesn't append a new line to the line before calling the callback. BoringSSL doesn't write to a log file at all (as OpenSSL doesn't here). Chrome does write the file, and it handles appending the new line.